test/extended/s2i_ruby_test.go

```diff
@@ -0,0 +1,81 @@
+// +build s2i
+
+package extended
+
+import (
+	"fmt"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	kapi "github.com/GoogleCloudPlatform/kubernetes/pkg/api"
+	exutil "github.com/openshift/origin/test/extended/util"
+)
+
+var _ = Describe("OpenShift Ruby S2I image", func() {
+	defer GinkgoRecover()
+	var (
+		oc       = exutil.NewCLI("mysql-create", kubeConfigPath())
+		versions = map[string]string{
+			"20-centos7": "ruby 2.0.0",
+			//"20-rhel7":   "ruby 2.0.0",
+			//"22-centos7": "ruby 2.2.2",
+			//"22-rhel7":   "ruby 2.2.2",
+		}
+	)
+
+	for v, expected := range versions {
+
+		Describe(fmt.Sprintf("SCL usage of ruby-%s", v), func() {
+			It("should provide usage", func() {
+				oc.SetOutputDir(testContext.OutputDir)
+				name := exutil.MustGetImageName(oc.REST().ImageStreams("openshift"), "ruby", "latest")
```

Comment:
what registered this imagestream?  how do we know it's referencing the image we want instead of something it found on dockerhub?  (this will be a bigger problem with v2 registries).  

I think the GetPodForContainer should use a straight docker pullspec that we know can be resolved locally if the image was built on this machine.